docs/internal/Filters-for-Applications.md

```diff
@@ -1,7 +1,22 @@
+## Contents <!-- omit in toc -->
+<!-- toc -->
+- [Overview](#overview)
+- [Examples: UI Design](#examples-ui-design)
+- [Filters](#filters)
+  - [Label filters:](#label-filters)
+  - [applicant action](#applicant-action)
```

Comment:
Don't change it here, but in the place where its used -- should be capitalised "Applicant action" for consistency with the others.